python/Ganga/GPIDev/Base/Objects.py

```diff
@@ -784,8 +788,8 @@ def __init__(self):
             for attr, item in self._schema.allItems():
                 ## If an object is hidden behind a getter method we can't assign a parent or defvalue so don't bother - rcurrie
                 if item.getProperties()['getter'] is None:
-                    defVal = self._schema.getDefaultValue(attr)
-                    self.setNodeAttribute(attr, defVal)
+                    setattr(self, attr, self._schema.getDefaultValue(attr))
```

Comment:
setNodeAttribute only sets the parent for values that are GangaObjects which fails (at least) when you have a default value of `[]` for a schema item as through the setter, this would get changed to a GangaList and the parent set correctly, but through setNodeAttribute does not. This then causes the `print_summary` to be ignored as well as other possible issues. There may be other cases where not running through the setter causes problems but this was the reason I put this change in.